src/modules/submitScore.js

```diff
@@ -1,27 +1,23 @@
-const API_URL =  "https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xHcNAgk6UkMi3fEdct8S/scores";
+const API_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xHcNAgk6UkMi3fEdct8S/scores';
+const userName = document.getElementById('userName');
+const userScore = document.getElementById('userScore');
 
-function postMessage(data) {
+class Data {
+  'user':"userName.value"
+
+  'score': "userName.value"
+}
+
+function postMessage() {
   const config = {
     method: 'POST',
     headers: {
-      'Accept': 'application/json',
-      'Content-Type': 'application/json'
+      'Access-Control-Allow-Origin': 'origin',
+      'Content-Type': 'application/json',
     },
-    body: JSON.stringify(data)
-  }
-  fetch(API_URL, config)
-  .then((r) => r.json())
-  .then((data) => {
-    console.log(data)
-  })
```

Comment:
- [ ] It would be nice to use `async and await` functions instead of `promise` as specified in the project requirements.
The syntax for async and await looks like this:

```
const postMessage = async () => {
  const response = await fetch(URL, {
    method: 'POST',
    body: JSON.stringify({ user: userName, score: userScores }),
    headers: {
      'Content-type': 'application/json; charset=UTF-8',
    },
  });
  const data = await response.json();
  // then code for what you want to do with the data
};

```

- [ ] Another requirement is to use _**ES6**_ syntax for initializing a function. Kindly refer to the way I initialized `postMessage` function in the code snippet above.
 

Kindly implement them 👍